spark/src/test/java/org/apache/iceberg/actions/TestExpireSnapshotsAction.java

```diff
@@ -1053,7 +1055,7 @@ public void testUseLocalIterator() {
     int jobsAfter = spark.sparkContext().dagScheduler().nextJobId().get();
     int totalJobsRun = jobsAfter - jobsBefore;
 
-    checkExpirationResults(1L, 1L, 2L, results);
+    checkExpirationResults(1L, 0L, 1L, 2L, results);
```

Comment:
I don't see a test that actually tests delete file count > 0, could you add one?

Reply:
Thanks, I will add the unit tests.